packages/critters-webpack-plugin/src/index.js

```diff
@@ -0,0 +1,243 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import path from 'path';
+import minimatch from 'minimatch';
+import sources from 'webpack-sources';
+import log from 'webpack-log';
+import Critters from 'critters';
```

Comment:
`import Critters from '../../../src'` did not work

Reply:
Indeed - it's disallowed by the `"files"` option and would break when packaged.